spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -16,8 +16,9 @@
 
 package org.springframework.cloud.gcp.storage;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import java.util.logging.Logger;
```

Comment:
Completely agree regarding JUL. It's horrible!
@artembilan Looking at boot documentation, it seems like Logback is preferred. Should we be using that?